pipeline_documentation.md

```diff
@@ -1000,11 +1000,11 @@ OUT:
 Filter multimappers favoring mismatches over InDels with a
 [**custom script**][custom-script-filter-mm].
 
-> Under the assumption that InDels are less frequent than mismatches only
-> those alignments (of the same read with the same edit distance) with the
-> lowest number of InDels are kept. This approach allows the presence of
-> multimappers and/or InDels after the filtering if the alignments contain the
-> same proportion of mismatches vs. InDels.
+> Provided that InDels are more frequent than mismatches only those alignments
```

Comment:
Use "Given" instead of "Provided" and explicitly mention that this refers to miRNAs. Also include the references to the papers that show this.


> Given that InDels are more frequent in miRNAs than mismatches, as demonstrated by [Saunders _et al._ (2017)](https://pmc.ncbi.nlm.nih.gov/articles/PMC1805605/), OTHER REF, and OTHER REF, only those "multimappers" (defines here as alignments of the same read to different genomic loci with the same edit distance) are retained that contain a higher number of InDels compared to mismatches.

You can then remove the following sentence as it is a bit confusing.